lms/static/sass/shared-v2/_modal.scss

```diff
@@ -11,261 +11,259 @@
-                                                          rgba(255,255,255, 0)));
-        height: 1px;
-        width: 100%;
-        border: none;
-        margin: 0;
+    display: none;
+    position: absolute;
+    left: 50%;
+    padding: 8px;
+    border-radius: 3px;
+    box-shadow: 0 0px 5px 0 $shadow-d1;
+    background: $gray-d2;
+    color: $base-font-color;
+
+    .inner-wrapper {
+        @include padding(0, $baseline/2, $baseline/2, $baseline/2);
+        z-index: z-index(mid-front);
+        background: $modal-bg-color;
+        border-radius: 0;
+        border: 1px solid $lms-border-color;
+        box-shadow: inset 0 1px 0 0 rgba(255, 255, 255, 0.7);
+        overflow: hidden;
         position: relative;
-        z-index: 2;
-
-        &::after {
-          @include background-image(linear-gradient(180deg, rgba(200,200,200, 0) 0%,
-                                                            rgba(200,200,200, 1) 50%,
-                                                            rgba(200,200,200, 0)));
-          height: 1px;
-          width: 100%;
-          bottom: 0;
-          content: '';
-          display: block;
-          position: absolute;
-          top: -1px;
-        }
-      }
 
-      h2 {
-        @extend .hd-4;
-        position: relative;
-        text-align: center;
-        text-shadow: 0 1px rgba(255,255,255, 0.4);
-        z-index: 2;
-        text-transform: uppercase;
-        font-family: $font-family-serif;
-
-        .edx {
-          text-transform: none;
+        p {
+            font-size: font-size(small);
+            line-height: 1.4;
         }
-      }
-    }
 
-    .modal-form-error {
-      background: tint($red,90%);
-      border: 1px solid rgb(143, 14, 14);
-      color: rgb(143, 14, 14);
-      display: none;
-      margin-bottom: $baseline;
-      padding: 12px;
-    }
-
-    .notice {
-      background: $yellow;
-      border: 1px solid darken($yellow, 60%);
-      color: darken($yellow, 60%);
-      display: none;
-      margin-bottom: $baseline;
-      padding: 12px;
-    }
+        a {
+            &:hover,
+            &:focus {
+                text-decoration: underline;
+            }
+        }
 
-    .activation-message, .message {
-      padding: 0 ($baseline*2) ($baseline/2);
+        header {
+            z-index: z-index(mid-front);
+            margin-bottom: ($baseline*1.5);
+            overflow: hidden;
+            padding: 28px $baseline 0;
+            position: relative;
+
+            &::before {
+                @include background-image(radial-gradient(50% 50%, circle closest-side, rgba(255,255,255, 0.8) 0%, rgba(255,255,255, 0) 100%));
+                content: "";
+                display: block;
+                height: 400px;
+                left: 0;
+                margin: 0 auto;
+                position: absolute;
+                top: -140px;
+                width: 100%;
+                z-index: 1;
```

Comment:
I believe that you wrote all of the Sass in this file. I just indented it. I'd rather not clean it all up as part of this PR if that's okay.